projects/saturn/src/main/java/io/fairspace/saturn/controller/UserController.java

```diff
@@ -0,0 +1,36 @@
+package io.fairspace.saturn.controller;
+
+import java.util.Collection;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import io.fairspace.saturn.services.users.User;
+import io.fairspace.saturn.services.users.UserRolesUpdate;
+import io.fairspace.saturn.services.users.UserService;
+
+@RestController
+@RequestMapping("/api/users/")
+@RequiredArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+
+    @GetMapping
```

Comment:
The annotations are not consistent in all controllers, especially for getters - sometimes you specify only `@GetMapping`, sometimes it's more extensive including: `@GetMapping(value = "/", produces = MediaType.APPLICATION_JSON_VALUE)`

Reply:
good point, aligned with one style